modules/Emails/EmailUIAjax.php

```diff
@@ -996,12 +996,29 @@
         break;
 
     case "setFolderViewSelection": // flows into next case statement
+        global $db;
         $GLOBALS['log']->debug("********** EMAIL 2.0 - Asynchronous - at: setFolderViewSelection");
         $viewFolders = $_REQUEST['ieIdShow'];
+        $userSubscriptions = $email->et->folder->getSubscriptions($focus);
```

Comment:
Admins should be able to set which folders for a user. In order to achieve this we need to pass the user that is being edited. I have added the lines required.